### ISSUE TITLE
[with solution]: problem with lineWidthStack

### ISSUE THREAD

**Issue by tmarti** [hustcc/jest-canvas-mock#53]
This line...

https://github.com/hustcc/jest-canvas-mock/blob/d89582496b40a66e490c6f37d4faec6ffce6bd85/src/classes/CanvasRenderingContext2D.js#L1116

... should be replaced with:

```javascript
this._lineWidthStack.push(this._lineWidthStack[this._stackIndex]); 
```

Notice the missing `_` when accessing `this._stackIndex`.

Sorry for not submitting a PR 😜

**Comment by jtenner** [comment]
What's really alarming is the lineWidth isn't being properly tested.

**Comment by jtenner** [comment]
Thanks @tmarti  this should be fixed in #52
